index.html

```diff
@@ -58,40 +58,74 @@ <h3>2022.11.21(MON) ~ 25 (FRI)</h3>
+                    </p>
+                </div>
+                <div class=" container-fluid programOptions">
+                    <div class="initial"><img src="./icons/exhibition.png" alt="Exhibition Icon"></div>
+                    <p class="middle">NFT Exhibition</p>
+                    <p class="final">Listen to the speakers from various countries about the messeges of sahrin and opening.
+                    </p>
+                </div>
+                <div class=" container-fluid programOptions">
+                    <div class="initial"><img src="./icons/chat.png" alt="Forum Icon"></div>
+                    <p class="middle">Forum</p>
+                    <p class="final">Listen to the speakers from various countries about the messeges of sahrin and opening.
+                    </p>
+                </div>
+                <div class=" container-fluid programOptions">
+                    <div class="initial"><img src="./icons/workshop.png" alt="Workshop Icon"></div>
+                    <p class="middle">Workshop</p>
+                    <p class="final">Listen to the speakers from various countries about the messeges of sahrin and opening.
+                    </p>
+                </div>
+                <div class=" container-fluid programOptions">
+                    <div class="initial"><img src="./icons/insight.png" alt="Insight Icon"></div>
+                    <p class="middle">Insight</p>
+                    <p class="final">Listen to the speakers from various countries about the messeges of sahrin and opening.
+                    </p>
+                </div>
             </div>
             <div class="join"><button type="button" class="btn btn-danger">Join NFT Art Gallery</button></div>
+            <div class="linkDesktop"><a href="#">SEE THE WHOLE PROGRAM</a> </div>
         </div>
         <div class="container-fluid bg-light speakers">   
         </div>
+        <div class="container-fluid bg-dark partnerHome">
+            <h3>Partner</h3>
+            <div class="separator"></div>
+            <div class="row">
+                <div class="col">
+                  <img src="./logos/firefox.jpeg" alt="Mozilla firefox">
+                </div>
+                <div class="col">
+                  <img src="./logos/google.jpg" alt="Google">
+                </div>
+                <div class="col">
+                  <img src="./logos/naver.png" alt="Naver">
+                </div>
+                <div class="col">
+                  <img src="./logos/daum.png" alt="Daumkakao">
+                </div>
+                <div class="col">
+                  <img src="./logos/airbnb.png" alt="AirBnb">
+                </div>
+            </div>     
+        </div>
+        <div class="container-fluid footeresHome">
+            <div class="row">
+                <div class="col">
+                  <img src="./images/creativecommons.png" alt="Creative Commons">
+                </div>
```

Comment:
- You have added the `creative commons korea` as an image What about writing it out as a text instead?? Maybe a `<p>` tag would do 🤔 

Here is what I can see 👇🏽 

| Your footer  |That of Behance |
| :---:   | :---: | 
| ![footbad](https://user-images.githubusercontent.com/99246359/203656681-ac4b588b-bb27-4e6f-adbd-c28d7158f53d.PNG) |![footgood](https://user-images.githubusercontent.com/99246359/203656770-c09dfb76-019f-47a2-9b3e-028d33d79c39.PNG)  |

Why don't we remove the image and use text instead? Or, what do you think? 💭